clang/test/Driver/mips-msa.c

```diff
@@ -0,0 +1,8 @@
+// RUN: %clang -### -c --target=mips64el-unknown-linux-gnuabi64 \
```

Comment:
We don't need -fno-integrated-as tests. They are just forwarded.

The -mmsa test needs a NOT. I'll push a commit to improve the test

Reply:
OK,Thanks!